Description: incorrectly strips whitespace (patch included)
 In the previous upload, docbook2man was patched to escape the special
 characters . and ' at the beginning of a line (see #399947).
 Unfortunately that patch was incorrect -- as well as escaping the special
 characters, it also removes all whitespace from the beginning of
 affected lines.  As a result, if you are in a block where whitespace
 matters (such as <screen>...</screen>) then the output is now incorrect.
 
 This is very easy to fix -- just match the whitespace and preserve it,
 instead of tossing it away.  The full patch (just two lines) is included
 below.
Author: Ben Burton <bab@debian.org>
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/528334

Index: docbook-utils-0.6.14/helpers/docbook2man-spec.pl
===================================================================
--- docbook-utils-0.6.14.orig/helpers/docbook2man-spec.pl	2012-05-09 18:59:42.584774885 +0200
+++ docbook-utils-0.6.14/helpers/docbook2man-spec.pl	2012-05-09 19:00:34.788773002 +0200
@@ -1212,8 +1212,8 @@
 	$_[0] =~ s/\\/\\\\/g;
 
 	# Escape dots and single quotes in column 1
-	$_[0] =~ s/^[ \t]*\./\\\&\./;
-	$_[0] =~ s/^[ \t]*\'/\\\&\'/;
+	$_[0] =~ s/^([ \t]*)\./$1\\\&\./;
+	$_[0] =~ s/^([ \t]*)\'/$1\\\&\'/;
 
 	# In non-'pre'-type elements:
 	if(!$nocollapse_whitespace) {
